src/MSI/Product.wxs

```diff
@@ -111,6 +111,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
                    <File Source="..\AccessibilityInsights.Extensions.Telemetry\bin\Release\AccessibilityInsights.Extensions.Telemetry.dll" Id="telemetry_extension"/>
                    <File Source="..\AccessibilityInsights.Extensions.Telemetry\bin\Release\AccessibilityInsights.Extensions.Telemetry.dll.config" />
                    <File Source="..\AccessibilityInsights.Extensions.Telemetry\bin\Release\Microsoft.ApplicationInsights.dll" Id="applicationinsights_extension" />
+                   <File Source="..\AccessibilityInsights.Extensions.Telemetry\bin\Release\System.Diagnostics.DiagnosticSource.dll" Id="diagnosticsource_extension" />
```

Comment:
This file was referenced in the previous version of the assembly but was incorrectly omitted from the wxs file. It's apparently not a big deal that it's missing (since the telemetry is working), but I added it for completeness